Tests/Clappr_Tests/Classes/Base/Layers/ContainerLayerTests.swift

```diff
@@ -0,0 +1,39 @@
+import Quick
+import Nimble
+@testable import Clappr
+
+class ContainerLayerTests: QuickSpec {
+    override func spec() {
+        describe(".ContainerLayer") {
+            context("When a Container is attached") {
+                it("resizes to match the layer bounds") {
+                    let containerView = UIView(frame: .zero)
+                    let frame = CGRect(x: 0, y: 0, width: 100, height: 100)
+                    let layer = ContainerLayer(frame: frame)
+                    
+                    layer.attachContainer(containerView)
+                    layer.layoutIfNeeded()
+                    
+                    expect(containerView.center).to(equal(layer.center))
+                    expect(containerView.bounds.size).to(equal(layer.bounds.size))
+                    
```

Comment:
I think that we can change for explicit values and not related objects inside the test arrange, once we can change the test arrange or the ContainerLayer code and the code will not reflect the changes or cover the main behavior. What do you think about it?

Reply:
Solved on 9614ccf02307e4d9d30380cdaf81a0471999f494